cb-tools/burgbox/+io/DummyCommunicator.m

```diff
@@ -8,10 +8,18 @@
 
   % 2013-03 CB created  
   
+  properties
```

Comment:
Where is this class used? Is it used in anything not in a test?

Reply:
following up on this

Reply:
This is used in a test on another branch that is not yet finished.